langchain/document_loaders/base.py

```diff
@@ -44,3 +45,44 @@ def lazy_load(
         raise NotImplementedError(
             f"{self.__class__.__name__} does not implement lazy_load()"
         )
+
+
+class BaseBlobParser(abc.ABC):
+    """Abstract interface for blob parsers.
+
+    A blob parser is provides a way to parse raw data stored in a blob into one
+    or more documents.
+
+    The parser can be composed with blob loaders, making it easy to re-use
+    a parser independent of how the blob was originally loaded.
+    """
+
+    @abc.abstractmethod
```

Comment:
believe abc and abstractmethod are explicitly imported, so no need for abc.

Reply:
lol this keeps on happening to me :)

Reply:
I'll follow up with a clean up in a second.